packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -258,35 +258,20 @@ export class Meter implements api.Meter {
 
   /**
    * Creates a new batch observer metric.
-   * @param name the name of the metric.
    * @param callback the batch observer callback
    * @param [options] the metric batch options.
    */
   createBatchObserver(
-    name: string,
     callback: (observerResult: api.BatchObserverResult) => void,
     options: api.BatchMetricOptions = {}
-  ): api.BatchObserver {
-    if (!this._isValidName(name)) {
-      this._logger.warn(
-        `Invalid metric name ${name}. Defaulting to noop metric implementation.`
-      );
-      return api.NOOP_BATCH_OBSERVER_METRIC;
-    }
+  ) {
     const opt: api.BatchMetricOptions = {
       logger: this._logger,
       ...DEFAULT_METRIC_OPTIONS,
```

Comment:
Should this still use the `DEFAULT_METRIC_OPTIONS` as this is not metric now right ?